cmd/virt-launcher/virt-launcher.go

```diff
@@ -335,6 +338,20 @@ func waitForFinalNotify(deleteNotificationSent chan watch.Event,
 	}
 }
 
+func runDownwardMetricsVirtioServer(ctx context.Context) error {
+	if _, ok := os.LookupEnv(services.ENV_VAR_VIRT_LAUNCHER_DOWNWARDMETRICS_SERVER); ok {
+		nodeName, ok := os.LookupEnv(services.ENV_VAR_NODE_NAME)
+		if !ok {
+			return errors.New(services.ENV_VAR_NODE_NAME + " environment variable not found")
```

Comment:
I'm not sure if I understand correctly. I would expect to exit early without an error if there is no need to run the server, but if there is a need, and we failed to start, we should exit.

Reply:
Done.